### PR TITLE
Doomsday highlight fixes

### PR DESCRIPTION
**Pull request by nathanstitt**

### CODE REVIEW

**Comment by nathanstitt** [comment]
[Bug: Highlighting Bugs from Doomsday for January 16](https://trello.com/c/nKZ1m290/1153-bug-highlighting-bugs-from-doomsday-for-january-16)

**Comment by nathanstitt** [comment]
[Bug: Highlighting Bugs from Doomsday for January 16](https://trello.com/c/nKZ1m290/1153-bug-highlighting-bugs-from-doomsday-for-january-16)

**Comment by nathanstitt** [comment]
[T3.highlighting - Improvements to the Summary Page](https://trello.com/c/7aNbNqcm/1151-t3highlighting-improvements-to-the-summary-page)

File: tutor/src/helpers/dom.js [hunk 1]
```diff
@@ -0,0 +1,277 @@
+/**
+ * Utility functions to make DOM manipulation easier.
+ * @param {Node|HTMLElement} [el] - base DOM element to manipulate
+ * @returns {object}
+ */
```
Comment (pandafulmanda) [inline tutor/src/helpers/dom.js]: nice collection

**Pull request merged**